### PR TITLE
allow customCSS to reorder/hide disabled scripts in popup

### PR DESCRIPTION
**Pull request by tophf**
With this PR customCSS can reorder/hide disabled scripts in popup by setting `flex` on `.submenu` and `order` or `display` on `.submenu .disabled`.

Ideally I would like to implement this as a user option in the dashboard but you seem to prefer the minimalistic built-in set of options, right?

### CODE REVIEW

**Comment by gera2ld** [comment]
The execution order of scripts can be resorted by dragging, which requires each script item to have the same height, and the display order must be the same as that in the JS list. I'm afraid this change will break the resorting thing.

BTW, currently we support sorting in alphabetical order and also support showing enabled scripts first.
image

It's OK to add a new option if it does not require the user to make a choice.

**Comment by tophf** [comment]
This PR is for the popup, not the dashboard. 
The dashboard already puts the `.disabled` attribute on the entry itself so it's alright.

**Comment by tophf** [comment]
Also, I don't see why moving the class name to its proper place on the entry worsens anything. AFAICT it only improves things and allows for easier customizing of the UI.

**Comment by gera2ld** [comment]
My fault, I was looking into the wrong place.

> Also, I don't see why moving the class name to its proper place on the entry worsens anything. AFAICT it only improves things and allows for easier customizing of the UI.

This is reasonable.

File: src/popup/style.css [hunk 1]
```diff
@@ -37,7 +37,7 @@ footer {
     background: cornflowerblue;
     color: white;
   }
-  &.disabled {
+  .disabled & {
```
Comment (gera2ld) [inline src/popup/style.css]: Maybe stricter?
```css
.disabled > & {
```

Comment (tophf) [reply src/popup/style.css]: Yep.

**Pull request merged**